modules/Kconfig.nordic

```diff
@@ -10,121 +10,363 @@ config HAS_NORDIC_DRIVERS
 config HAS_NRFX
 	bool
 
-config NRFX_PRS
-	bool
+if HAS_NRFX
```

Comment:
Can put a `depends on` directly on the menu instead, like

```
menu "nrfx drivers"
	depends on HAS_NRFX

...
```

Reply:
Good to know, I'll update it.